openedx/features/edly/utils.py

```diff
@@ -333,3 +334,46 @@ def create_learner_link_with_permission_groups(user):
         user.groups.add(new_group)
 
     return user
+
+
+def get_current_site_invalid_certificate_context(default_html_certificate_configuration):
+    """
+    Gets current site's context data for invalid certificate.
+
+    Try to get current site's context data for invalid certificate from site configuration
+    or fallback to default html certificate configurations.
+
+    Arguments:
+        default_html_certificate_configuration (dict): Default html configurations dict.
+
+    Returns:
+        dict: Context data.
+    """
+    context = dict(default_html_certificate_configuration.get('default'))
+
+    current_site_configuration = get_current_site_configuration()
+
+    if not current_site_configuration:
+        return context
+
+    context['platform_name'] = current_site_configuration.get_value('platform_name', settings.PLATFORM_NAME)
+
+    branding = current_site_configuration.get_value('BRANDING', {})
+    context['logo_src'] = branding.get(
+        'logo',
+        default_html_certificate_configuration.get('default', {}).get('logo_src')
+    )
+
+    default_logo_url = default_html_certificate_configuration.get('default', {}).get('logo_url')
+    context['logo_url'] = settings.LMS_ROOT_URL if settings.LMS_ROOT_URL else default_logo_url
+
+    marketing_site_url = current_site_configuration.get_value('MKTG_URLS', {}).get('ROOT')
+    marketing_site_url = marketing_site_url[:-1] if marketing_site_url[-1] == '/' else marketing_site_url
```

Comment:
use LMS branding API for marketing URL's: `lms/djangoapps/branding/api.py`

Reply:
Done